### ISSUE TITLE
Centos 7 - cc-proxy requires selinux-policy version not available in Centos

### ISSUE THREAD

**Issue by chavafg** [clearcontainers/packaging#7]
When installing `cc-runtime`, `clear-containers-selinux` dependency fails to be installed since it looks for a`selinux-policy-base` and `selinux-policy-targeted` packages that is not available in CentOS.

```
Error: Package: clear-containers-selinux-0.1-1.1.noarch (home_clearcontainers_clear-containers-3-staging)
           Requires: selinux-policy-base >= 3.13.1-128.6.fc22
           Installed: selinux-policy-targeted-3.13.1-102.el7_3.16.noarch (installed)
               selinux-policy-base = 3.13.1-102.el7_3.16
           Available: selinux-policy-minimum-3.13.1-102.el7.noarch (base)
               selinux-policy-base = 3.13.1-102.el7
           Available: selinux-policy-minimum-3.13.1-102.el7_3.4.noarch (updates)
               selinux-policy-base = 3.13.1-102.el7_3.4
           Available: selinux-policy-minimum-3.13.1-102.el7_3.7.noarch (updates)
               selinux-policy-base = 3.13.1-102.el7_3.7
           Available: selinux-policy-minimum-3.13.1-102.el7_3.13.noarch (updates)
               selinux-policy-base = 3.13.1-102.el7_3.13
           Available: selinux-policy-minimum-3.13.1-102.el7_3.15.noarch (updates)
               selinux-policy-base = 3.13.1-102.el7_3.15
           Available: selinux-policy-minimum-3.13.1-102.el7_3.16.noarch (updates)
               selinux-policy-base = 3.13.1-102.el7_3.16
           Available: selinux-policy-mls-3.13.1-102.el7.noarch (base)
               selinux-policy-base = 3.13.1-102.el7
           Available: selinux-policy-mls-3.13.1-102.el7_3.4.noarch (updates)
               selinux-policy-base = 3.13.1-102.el7_3.4
           Available: selinux-policy-mls-3.13.1-102.el7_3.7.noarch (updates)
               selinux-policy-base = 3.13.1-102.el7_3.7
           Available: selinux-policy-mls-3.13.1-102.el7_3.13.noarch (updates)
               selinux-policy-base = 3.13.1-102.el7_3.13
           Available: selinux-policy-mls-3.13.1-102.el7_3.15.noarch (updates)
               selinux-policy-base = 3.13.1-102.el7_3.15
           Available: selinux-policy-mls-3.13.1-102.el7_3.16.noarch (updates)
               selinux-policy-base = 3.13.1-102.el7_3.16
           Available: selinux-policy-targeted-3.13.1-102.el7.noarch (base)
               selinux-policy-base = 3.13.1-102.el7
           Available: selinux-policy-targeted-3.13.1-102.el7_3.4.noarch (updates)
               selinux-policy-base = 3.13.1-102.el7_3.4
           Available: selinux-policy-targeted-3.13.1-102.el7_3.7.noarch (updates)
               selinux-policy-base = 3.13.1-102.el7_3.7
           Available: selinux-policy-targeted-3.13.1-102.el7_3.13.noarch (updates)
               selinux-policy-base = 3.13.1-102.el7_3.13
           Available: selinux-policy-targeted-3.13.1-102.el7_3.15.noarch (updates)
               selinux-policy-base = 3.13.1-102.el7_3.15
Error: Package: clear-containers-selinux-0.1-1.1.noarch (home_clearcontainers_clear-containers-3-staging)
           Requires: selinux-policy-targeted >= 3.13.1-128.6.fc22
           Installed: selinux-policy-targeted-3.13.1-102.el7_3.16.noarch (installed)
               selinux-policy-targeted = 3.13.1-102.el7_3.16
           Available: selinux-policy-targeted-3.13.1-102.el7.noarch (base)
               selinux-policy-targeted = 3.13.1-102.el7
           Available: selinux-policy-targeted-3.13.1-102.el7_3.4.noarch (updates)
               selinux-policy-targeted = 3.13.1-102.el7_3.4
           Available: selinux-policy-targeted-3.13.1-102.el7_3.7.noarch (updates)
               selinux-policy-targeted = 3.13.1-102.el7_3.7
           Available: selinux-policy-targeted-3.13.1-102.el7_3.13.noarch (updates)
               selinux-policy-targeted = 3.13.1-102.el7_3.13
           Available: selinux-policy-targeted-3.13.1-102.el7_3.15.noarch (updates)
               selinux-policy-targeted = 3.13.1-102.el7_3.15
```

**Comment by gorozco1** [comment]
We do not have `clear-containers-selinux` as a requeriment to run `cc-runtime` 

https://github.com/clearcontainers/packaging/blob/master/runtime/cc-runtime.spec-template#L19-L23

**Comment by chavafg** [comment]
Oh, sorry. It is not needed for `cc-runtime`, but needed by `cc-proxy`
I have updated the title of this issue.

This is the full output when trying to install `cc-proxy`:
```
[cloud@centos-obs-cc3 cc-oci-runtime]$ sudo yum -y install cc-proxy
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: bay.uchicago.edu
 * extras: centos.den.host-engine.com
 * updates: centos.chi.host-engine.com
Resolving Dependencies
--> Running transaction check
---> Package cc-proxy.x86_64 0:3.0.0alpha.1+git.41b7899-9.1 will be installed
--> Processing Dependency: qemu-lite for package: cc-proxy-3.0.0alpha.1+git.41b7899-9.1.x86_64
--> Processing Dependency: clear-containers-selinux for package: cc-proxy-3.0.0alpha.1+git.41b7899-9.1.x86_64
--> Processing Dependency: cc-proxy-config for package: cc-proxy-3.0.0alpha.1+git.41b7899-9.1.x86_64
--> Processing Dependency: cc-proxy-bin for package: cc-proxy-3.0.0alpha.1+git.41b7899-9.1.x86_64
--> Running transaction check
---> Package cc-proxy.x86_64 0:3.0.0alpha.1+git.41b7899-9.1 will be installed
--> Processing Dependency: qemu-lite for package: cc-proxy-3.0.0alpha.1+git.41b7899-9.1.x86_64
---> Package cc-proxy-bin.x86_64 0:3.0.0alpha.1+git.41b7899-9.1 will be installed
---> Package cc-proxy-config.x86_64 0:3.0.0alpha.1+git.41b7899-9.1 will be installed
---> Package clear-containers-selinux.noarch 0:0.1-1.1 will be installed
--> Processing Dependency: selinux-policy-targeted >= 3.13.1-128.6.fc22 for package: clear-containers-selinux-0.1-1.1.noarch
--> Processing Dependency: selinux-policy-base >= 3.13.1-128.6.fc22 for package: clear-containers-selinux-0.1-1.1.noarch
--> Finished Dependency Resolution
Error: Package: clear-containers-selinux-0.1-1.1.noarch (home_clearcontainers_clear-containers-3-staging)
           Requires: selinux-policy-base >= 3.13.1-128.6.fc22
           Installed: selinux-policy-targeted-3.13.1-102.el7_3.16.noarch (installed)
               selinux-policy-base = 3.13.1-102.el7_3.16
           Available: selinux-policy-minimum-3.13.1-102.el7.noarch (base)
               selinux-policy-base = 3.13.1-102.el7
           Available: selinux-policy-minimum-3.13.1-102.el7_3.4.noarch (updates)
               selinux-policy-base = 3.13.1-102.el7_3.4
           Available: selinux-policy-minimum-3.13.1-102.el7_3.7.noarch (updates)
               selinux-policy-base = 3.13.1-102.el7_3.7
           Available: selinux-policy-minimum-3.13.1-102.el7_3.13.noarch (updates)
               selinux-policy-base = 3.13.1-102.el7_3.13
           Available: selinux-policy-minimum-3.13.1-102.el7_3.15.noarch (updates)
               selinux-policy-base = 3.13.1-102.el7_3.15
           Available: selinux-policy-minimum-3.13.1-102.el7_3.16.noarch (updates)
               selinux-policy-base = 3.13.1-102.el7_3.16
           Available: selinux-policy-mls-3.13.1-102.el7.noarch (base)
               selinux-policy-base = 3.13.1-102.el7
           Available: selinux-policy-mls-3.13.1-102.el7_3.4.noarch (updates)
               selinux-policy-base = 3.13.1-102.el7_3.4
           Available: selinux-policy-mls-3.13.1-102.el7_3.7.noarch (updates)
               selinux-policy-base = 3.13.1-102.el7_3.7
           Available: selinux-policy-mls-3.13.1-102.el7_3.13.noarch (updates)
               selinux-policy-base = 3.13.1-102.el7_3.13
           Available: selinux-policy-mls-3.13.1-102.el7_3.15.noarch (updates)
               selinux-policy-base = 3.13.1-102.el7_3.15
           Available: selinux-policy-mls-3.13.1-102.el7_3.16.noarch (updates)
               selinux-policy-base = 3.13.1-102.el7_3.16
           Available: selinux-policy-targeted-3.13.1-102.el7.noarch (base)
               selinux-policy-base = 3.13.1-102.el7
           Available: selinux-policy-targeted-3.13.1-102.el7_3.4.noarch (updates)
               selinux-policy-base = 3.13.1-102.el7_3.4
           Available: selinux-policy-targeted-3.13.1-102.el7_3.7.noarch (updates)
               selinux-policy-base = 3.13.1-102.el7_3.7
           Available: selinux-policy-targeted-3.13.1-102.el7_3.13.noarch (updates)
               selinux-policy-base = 3.13.1-102.el7_3.13
           Available: selinux-policy-targeted-3.13.1-102.el7_3.15.noarch (updates)
               selinux-policy-base = 3.13.1-102.el7_3.15
Error: Package: clear-containers-selinux-0.1-1.1.noarch (home_clearcontainers_clear-containers-3-staging)
           Requires: selinux-policy-targeted >= 3.13.1-128.6.fc22
           Installed: selinux-policy-targeted-3.13.1-102.el7_3.16.noarch (installed)
               selinux-policy-targeted = 3.13.1-102.el7_3.16
           Available: selinux-policy-targeted-3.13.1-102.el7.noarch (base)
               selinux-policy-targeted = 3.13.1-102.el7
           Available: selinux-policy-targeted-3.13.1-102.el7_3.4.noarch (updates)
               selinux-policy-targeted = 3.13.1-102.el7_3.4
           Available: selinux-policy-targeted-3.13.1-102.el7_3.7.noarch (updates)
               selinux-policy-targeted = 3.13.1-102.el7_3.7
           Available: selinux-policy-targeted-3.13.1-102.el7_3.13.noarch (updates)
               selinux-policy-targeted = 3.13.1-102.el7_3.13
           Available: selinux-policy-targeted-3.13.1-102.el7_3.15.noarch (updates)
               selinux-policy-targeted = 3.13.1-102.el7_3.15
Error: Package: cc-proxy-3.0.0alpha.1+git.41b7899-9.1.x86_64 (home_clearcontainers_clear-containers-3-staging)
           Requires: qemu-lite
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
[cloud@centos-obs-cc3 cc-oci-runtime]$ 

```
